[Bug Report] nil pointer deference panic after upgrading to v0.16.1

Running v0.16.1 on Windows x64.

Attempting to launch Stash (just running stash-win.exe) results in an immediate crash:

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: Panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x20 pc=0x15d87e1]

goroutine 43 [running]:
main.handleError({0x210f280, 0xc000574140})
        github.com/stashapp/stash/cmd/stash/main.go:53 +0x93
main.recoverPanic()
        github.com/stashapp/stash/cmd/stash/main.go:44 +0x58
panic({0x1a70ea0, 0x19353f0})
        runtime/panic.go:1038 +0x215
github.com/stashapp/stash/internal/api.newImageBox.func1({0x20ee4f8, 0x1}, {0x0, 0x0}, {0x0, 0x124855e})
        github.com/stashapp/stash/internal/api/images.go:35 +0x41
io/fs.WalkDir({0x2116640, 0xc000574120}, {0x20ee4f8, 0x1}, 0xc00069e588)
        io/fs/walk.go:110 +0x6a
github.com/stashapp/stash/internal/api.newImageBox({0x2116640, 0xc000574120})
        github.com/stashapp/stash/internal/api/images.go:34 +0x92
github.com/stashapp/stash/internal/api.initialiseCustomImages()
        github.com/stashapp/stash/internal/api/images.go:76 +0xfc
github.com/stashapp/stash/internal/api.initialiseImages()
        github.com/stashapp/stash/internal/api/images.go:67 +0x17d
github.com/stashapp/stash/internal/api.Start()
        github.com/stashapp/stash/internal/api/server.go:45 +0x36
main.main.func1()
        github.com/stashapp/stash/cmd/stash/main.go:29 +0x3f
created by main.main
        github.com/stashapp/stash/cmd/stash/main.go:27 +0x5e
```

I was previously on 0.15.

--- a/internal/api/images.go
+++ b/internal/api/images.go
@@ -32,6 +32,10 @@ func newImageBox(box fs.FS) (*imageBox, error) {
 	}
 
 	err := fs.WalkDir(box, ".", func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+
 		if d.IsDir() {
 			return nil
 		}

